test/1_stdlib/FloatingPoint.swift.gyb

```diff
@@ -370,6 +370,15 @@ FloatingPoint.test("${Self}.addingProduct") {
   let ulp = ${Self}.ulpOfOne
   expectEqual(ulp*ulp, one.addingProduct(ulp + one, ulp - one))
 }
+
+FloatingPoint.test("${Self}/LosslessStringConvertible") {
+  let instances = [ 1.0, -1.0,
```

Comment:
This is temporary.  I sent Steven Canon an email asking for interesting instances to put here.